pkg/operator/ceph/cr_manager.go

```diff
@@ -156,11 +158,17 @@ func (o *Operator) startCRDManager(context context.Context, mgrErrorCh chan erro
 
 	// Set up a manager
 	mgrOpts := manager.Options{
-		LeaderElection:     false,
-		Namespace:          o.config.NamespaceToWatch,
-		MetricsBindAddress: "0",
-		Scheme:             scheme,
-		CertDir:            certDir,
+		LeaderElection: false,
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: {}},
+		},
+		Metrics: metricsserver.Options{
+			BindAddress: "0",
```

Comment:
Would you mind adding a small doc explaining what "0" means? I assume that means there is no bind addr?